viewer/core/src/datamodels/cad/rendering/EffectRenderManager.ts

```diff
@@ -186,7 +186,7 @@ export class EffectRenderManager {
     this._customObjectRenderTarget.depthTexture.format = THREE.DepthFormat;
     this._customObjectRenderTarget.depthTexture.type = THREE.UnsignedIntType;
 
-    this._compositionTarget = new THREE.WebGLRenderTarget(0, 0, { stencilBuffer: false });
+    this._compositionTarget = new THREE.WebGLRenderTarget(0, 0, { stencilBuffer: false }); // Final depth buffer
```

Comment:
Not sure if this comment is correct / makes sense. This rendertarget outputs a color-buffer as well.